aspnetcore/razor-pages/sdk.md

```diff
@@ -16,26 +16,21 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 The [!INCLUDE[](~/includes/2.1-SDK.md)] includes the `Microsoft.NET.Sdk.Razor` MSBuild SDK (Razor SDK). The Razor SDK:
 
-::: moniker range=">= aspnetcore-2.1 <= aspnetcore-2.2"
-* Standardizes the experience around building, packaging, and publishing projects containing [Razor](xref:mvc/views/razor) files for ASP.NET Core MVC-based projects.
-* Includes a set of predefined targets, properties, and items that allow customizing the compilation of Razor files.
-::: moniker-end
-
 ::: moniker range=">= aspnetcore-3.0"
-* is required to build, package, and publish projects containing [Razor](xref:mvc/views/razor) files for ASP.NET Core MVC-based projects, or [Blazor](xref:blazor/index) projects
-* Includes a set of predefined targets, properties, and items that allow customizing the compilation of Razor (.cshtml or .razor) files.
-::: moniker-end
-
 
-::: moniker range=">= aspnetcore-2.1 <= aspnetcore-2.2"
+* Is required to build, package, and publish projects containing [Razor](xref:mvc/views/razor) files for ASP.NET Core MVC-based or [Blazor](xref:blazor/index) projects.
+* Includes a set of predefined targets, properties, and items that allow customizing the compilation of Razor (*.cshtml* or *.razor*) files.
 
-The Razor SDK includes a `<Content>` element with an `Include` attribute set to the `**\*.cshtml` globbing pattern. Matching files are published.
+The Razor SDK includes `Content` MSBuild targets with `Include` attributes set to the `**\*.cshtml` and `**\*.razor` globbing patterns. Matching files are published.
```

Comment:
> The Razor SDK includes `Content` MSBuild targets

It's not a MSBuild target though. Content items maybe?

Reply:
Gotcha ... thanks.